ui-spacetimechart/src/components/workSchedules/Region.tsx

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useDraw } from '../../hooks/useCanvas';
+import { type DrawingFunction } from '../../lib/types';
+import { type SpaceTimeRegion } from '../../types';
+
+const Region = ({
+  timeStart, // x date timestamp
+  timeEnd, // x date timestamp
+  spaceStart, // y metre
+  spaceEnd, // y metre
```

Comment:
Remove this comment since the chart can be rotated
```suggestion
  timeStart,
  timeEnd,
  spaceStart,
  spaceEnd,
```

Reply:
done